src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        File fileRead = new File(fromFileName);
+        File fileWrite = new File(toFileName);
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fileRead));
+                BufferedWriter writer = new BufferedWriter(new FileWriter(fileWrite))) {
+
+            int supplySum = 0;
```

Comment:
The variables 'supplySum' and 'buySum' should be local variables within the try block since they are not used outside of it.